### ISSUE TITLE
Using Arbitriaries outside jqwik execution

### ISSUE THREAD

**Issue by Danny02** [jlink/jqwik#125]
## Testing Problem

In my project I use JUnit5 alongside jqwik. Until Version 1.2.7 I could use Arbitrary samples in JUnit tests. With version 1.3.6 I get the error "The current action must be run on a jqwik thread ..." in CurrentTestDescriptor.get(), which is called in the initialization of the EdgeCaseFacadeImpl.

I know that I could replace all @Test with @Example, but JUnit has some other features one would want to use. And also most coworkers are more acustomed to using JUnit.

**Comment by jlink** [comment]
@Danny02 Can you give an example how you use it? It _should_ still work...

**Comment by Danny02** [comment]
@jlink I created a minimal failing example for you
https://github.com/Danny02/jqwik-errors/blob/master/src/test/java/de/UseSampleTest.java

**Comment by jlink** [comment]
Copying over the example to have it handy:

```
public class UseSampleTest {

    @Test
    void useSample() {
        // strings uses EdgeCasesFacadeImpl, i.e. ints() does not
        Arbitraries.strings().sample();
    }
}
```

**Comment by jlink** [comment]
It's definitely a bug. 

Strangely enough I have tests for it, but they seem to work fine in the overall suite and only fail when run individually :-(

**Comment by jlink** [comment]
Fixed and published in 1.3.7-SNAPSHOT: https://jqwik.net/release-notes.html#137-snapshot
